app/models/route.rb

```diff
@@ -69,4 +70,14 @@ def self.send_email(routes)
     end
   end
 
+  def get_authority_info
+    latitude = self.waypoints.first.latitude
+    longitude = self.waypoints.first.longitude
+    response = Faraday.get('https://hifu-authorities-service.herokuapp.com/sheriff') do |req|
+      req.params['lat'] = latitude
+      req.params['long'] = longitude
+    end
+    parsed = JSON.parse(response.body)
```

Comment:
I think this might be better abstracted into its own service class

Reply:
I can definitely do that!

Reply:
I left it in the model because that was how we did the other Faraday calls, but agree that this one is a little more complex and could be abstracted out :)